l10n/en/newsletter_form.ftl

```diff
@@ -35,3 +35,9 @@ newsletter-form-thanks = Thanks!
 newsletter-form-leave-this-field-empty = Leave this field empty.
 newsletter-form-yes = Yes
 newsletter-form-no = No
+multi-newsletter-form-checkboxes-legend = I want information about:
```

Comment:
l10n question: what are the options for prioritizing translations in certain languages?

we can add all 9 existing newsletter languages to mozilla-foundation & mozilla-and-you now, because confirmation emails exist in all of these languages:

- en
- de
- fr
- id
- es
- pl
- pt
- ru
- zh-TW

But if we want the multi-newsletter form to appear with all these languages, we have to wait for the translations to become available in pontoon?

(new confirmation emails are coming in NL & IT as well, we will need to update these languages in Basket and have translations available for them when the confirmation emails are available)